transform/mattermost-analytics/models/marts/product/fct_active_users.sql

```diff
@@ -21,10 +21,18 @@ with metrics as (
     group by activity_date, server_id
 )
 select
-    *
+    -- Client telemetry
+    m.*,
+    -- Server activity
+    coalesce(sas.daily_active_users, 0) as server_daily_active_users,
+    coalesce(sas.monthly_active_user, 0) as server_monthly_active_users,
```

Comment:
`monthly_active_user` -> `monthly_active_users` here as well since you made the change in the other places.

Reply:
Nice catch. CI/CD failed because of that.